ci/test-readme/generate-install.sh

```diff
@@ -161,10 +161,45 @@ these dependencies.
 ## Table of Contents
 
 - [Fedora 30](#fedora-30)
+- [openSUSE (Leap)](#opensuse-leap)
+- [Ubuntu (18.04 - Bionic Beaver)](#ubuntu-1804---bionic-beaver)
+- [Ubuntu (16.04 - Xenial Xerus)](#ubuntu-1604---xenial-xerus)
+- [Debian (10 Buster)](#debian-10---buster)
+- [Debian (9 Stretch)](#debian-9---stretch)
+- [CentOS 7](#centos-7)
 END_OF_PREAMBLE
 
 readonly DOCKERFILES_DIR="${BINDIR}/../kokoro/install"
 
 echo
 echo "### Fedora (30)"
 "${BINDIR}/extract-install.sh" "${DOCKERFILES_DIR}/Dockerfile.fedora"
+
+echo
+echo "### openSUSE (Leap)"
+"${BINDIR}/extract-install.sh" "${DOCKERFILES_DIR}/Dockerfile.opensuse-leap"
+
+echo
+echo "### Ubuntu (18.04 - Bionic Beaver)"
+"${BINDIR}/extract-install.sh" "${DOCKERFILES_DIR}/Dockerfile.ubuntu-bionic"
+
+echo
+echo "### Ubuntu (16.04 - Xenial Xerus)"
+"${BINDIR}/extract-install.sh" "${DOCKERFILES_DIR}/Dockerfile.ubuntu-xenial"
+
+echo
+echo "### Debian (10 - Buster)"
+"${BINDIR}/extract-install.sh" "${DOCKERFILES_DIR}/Dockerfile.debian-buster"
+
+echo
+echo "### Debian (9 - Stretch)"
+"${BINDIR}/extract-install.sh" "${DOCKERFILES_DIR}/Dockerfile.debian-stretch"
+
+echo
+echo "### CentOS (8)"
```

Comment:
Was "CentOS 8" supposed to appear in the list above?

Reply:
Indeed, thanks!